tce/CHANGES

```diff
@@ -4,6 +4,8 @@
 Notable changes and features
 ----------------------------
 - Added hexadecimal output to PIG
+- Added HDB with register files and a basic ALU for Xilinx Series 7 devices.
+  Thanks to Guillermo Payá-Vayá for the contribution of the shifter.
```

Comment:
Please also add the affiliation and check if Guillermo was the only author. Seems the copyright notice says Stephan, so better thank "Stephan Nolting and Guillermo Payá-Vayá / IMS, Leibniz Univ. Hannover"